Recrutify/Recrutify.Services/Services/ScheduleService.cs

```diff
@@ -47,5 +70,30 @@ public async Task<ScheduleDTO> GetByDatePeriodForCurrentUserAsync(DateTime? date
             var schedules = await _scheduleRepository.GetByDatePeriodAsync(userId, date.Value, daysNum);
             return _mapper.Map<ScheduleDTO>(schedules);
         }
+
+        private IEnumerable<ScheduleCandidateInfoDTO> GetScheduleCandidateInfos(IEnumerable<CandidateDTO> candidates, IEnumerable<InterviewAppointmentDTO> interviewAppointmentDTOs, Guid projectId)
+        {
+            var scheduleCandidateInfos = new List<ScheduleCandidateInfoDTO>();
+            foreach (var candidate in candidates)
+            {
+                var projectResult = candidate.ProjectResults.FirstOrDefault(pr => pr.ProjectId == projectId);
+                var scheduleCandidateInfo = interviewAppointmentDTOs.Where(a => a.CandidateId == candidate.Id && a.IsAppointment == true).Any() ? _mapper.Map<ScheduleCandidateInfoDTO>(candidate)
+                   : new ScheduleCandidateInfoDTO();
+                if (scheduleCandidateInfo != null)
+                {
+                    scheduleCandidateInfo.ProjectResult = new ScheduleCandidateProjectResultDTO()
+                    {
+                        IsAssignedOnInterview = true,
+                        PrimarySkill = projectResult.PrimarySkill,
+                        ProjectId = projectResult.ProjectId,
+                        Status = StatusDTO.TechInterviewOneStep,
```

Comment:
Why tech interview one step? What if it is recruiter or tech interview second step ?